libclc/clc/lib/generic/integer/clc_abs.inc

```diff
@@ -6,7 +6,14 @@
 //
 //===----------------------------------------------------------------------===//
 
+#ifdef __CLC_GEN_S
```

Comment:
Does this define exist? I remember it downstream but haven't seen it upstream.

Reply:
> I'm surprised this didnt' fold in instcombine anyway?

You're right. Thanks for the insight. I have created a PR to fix instcombine: https://github.com/llvm/llvm-project/pull/165200